msteams-platform/concepts/bots/bots-overview.md

```diff
@@ -1,37 +1,54 @@
 ---
-title: Get started with bots
-description: Describes how to get started with bots in Microsoft Teams
-keywords: teams bots get started
+title: Develop bots
+description: Describes how to get started developing bots in Microsoft Teams
+keywords: teams bots development
 ---
 
-# Get started with bots for Microsoft Teams
+# Develop bots for Microsoft Teams
 
-Build and connect intelligent bots to interact with Microsoft Teams users naturally through chat. Or provide a simple commands-based bot, to be used as your "command-line" interface for your broader Teams app experience. You can even bring your existing bot from another channel and add Teams-specific support to make your experience shine. 
+Build and connect intelligent bots to interact with Microsoft Teams users naturally through chat. Or provide a simple commands-based bot, to be used as your "command-line" interface for your broader Teams app experience. You can make a notification-only bot, which can push information relevant to your users directly to them in a channel or direct message. You can even bring your existing bot from another channel and add Teams-specific support to make your experience shine.
```

Comment:
What do you mean by channel here: ' ... from another channel and ...'?

Reply:
The nomenclature here is...difficult. In the Bot Framework, a _channel_ is a connection between Bot Framework and an app such as Teams—quite different from a channel in Teams. (See [Core Concepts](https://docs.microsoft.com/en-us/bot-framework/overview-how-bot-framework-works#core-concepts) in the Bot Framework docs.)

Having said that, I think I've found a way to write around the issue.